lib/tapioca/compilers/symbol_table/symbol_generator.rb

```diff
@@ -21,6 +24,8 @@ def initialize(gem, indent = 0)
           @seen = Set.new
           @alias_namespace ||= Set.new
           @symbol_queue = T.let(symbols.sort.dup, T::Array[String])
+
+          gem.files.each { |path| YARD.parse(path.to_s, [], Logger::Severity::ERROR) }
```

Comment:
Is it really an error for Tapioca purpose?

```suggestion
          gem.files.each { |path| YARD.parse(path.to_s, [], Logger::Severity::WARN) }
```

Reply:
Using warn is the default and it prints a lot of stuff to the screen related to YARD finding undocumented classes. We need to turn it off or else the output looks horrible.